src/main/java/com/powsybl/network_hypothesis/GeneratorStartupAdderImpl.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network_hypothesis;
```

Comment:
Why not com.powsybl.network.hypothesis? I think we should avoid the _.

Reply:
I followed the convention of `com.powsybl.balances_adjustment` but I agree, I prefer `com.powsybl.network.hypothesis` too